### DISCUSSION TITLE
Is this project still active?

### DISCUSSION THREAD

**Discussion by matteobucci** [tcgoetz/GarminDB#189] [General]
Hello!

I found out about this project when I was looking for a way to export summaries of my Garmin Data and I must say I am very impressed with the project.

In order to run the project, I needed to make some amendments to fix several errors that appeared during my usage. I'm not a python expert, but I would like to contribute to the project.

What is its current status? What is the best way I can contribute to it?

**Comment by tcgoetz** [comment]
The project is still maintained, but I haven't been working on it much during the summer. Please make a pull request against the develop branch with your fixes.

**Comment by matteobucci** [comment]
Thank you @tcgoetz!
One more question: is there a way to run the project without building from the sources every time? When making some changes, is there a quick way to debug them?

**Comment by tcgoetz** [comment]
What do you mean by building the sources? When I make some changes and want to test them, I do something like `make clean reinstall all test` Thats a pretty comprehensive test that takes a few minutes to run. Before I commit a big chnage I run a regression test with `make clean reinstall_all regression_test`.
